cmd/state-exec/main.go

```diff
@@ -90,6 +90,7 @@ func run() error {
 
 	logr.Debug("cmd - running: %s", meta.MatchingBin)
 	if err := runCmd(meta); err != nil {
+		logr.Debug("      running - failed: bins (%v)", meta.ExecMeta.Bins)
```

Comment:
This seems like a local debugging statement. If you want to commit it can you phrase it in a way that others reading it can understand what it's saying?

Reply:
The message is additive to the previous debug, so it should read nicely.